lardataobj/Simulation/SimEnergyDepositLite.h

```diff
@@ -0,0 +1,102 @@
+//////////////////////////////////////////////////////////////////////
+/// \file  lardataobj/Simulation/SimEnergyDepositLite.h
+/// \brief contains information for a single step in the detector simulation
+///					(pared down in size to the essential information)
+/// \authors Laura Domine
+////////////////////////////////////////////////////////////////////////
+#ifndef LARDATAOBJ_SIMULATION_SIMENERGYDEPOSITLITE_H
+#define LARDATAOBJ_SIMULATION_SIMENERGYDEPOSITLITE_H
+
+// LArSoft includes
+// Define the LArSoft standard geometry types and methods.
+#include "larcoreobj/SimpleTypesAndConstants/geo_vectors.h"
+#include "lardataobj/Simulation/SimEnergyDeposit.h"
+
+// C++ includes
+#include <vector>
+
+namespace sim
+{
+  /**
+   * @brief Energy deposition in the active material (lite version).
+   *
+   * The detector simulation (presently LArG4, which invokes Geant4)
+	 * propagates particles through the detector in intervals of "steps".
+	 * These are usually stored in sim::SimEnergyDeposit objects. For
+	 * filesize concerns we pare it down to the essential information.
+	 */
+	class SimEnergyDepositLite
+	{
+	public:
+
+		using Length_t = float;
+		using Point_t = geo::Point_t;
+
+		SimEnergyDepositLite(double e = 0.,
+												geo::Point_t middle = {0., 0., 0.},
+												double t = 0.,
+												int id = 0)
+											: edep(e)
+											, middlePos(middle)
+											, middleTime(t)
+											, trackID(id)
+		{}
+
+		double Energy() const { return edep; }
+		geo::Point_t Position() const { return { middlePos.X(), middlePos.Y(), middlePos.Z() }; }
+    geo::Point_t MidPoint() const { return Position();} ///< Just an alias for compatibility with SED
+		double Time() const { return middleTime; }
+		int TrackID() const { return trackID; }
+		void setTrackID(int id) { trackID = id; }
+		void setEnergy(double e) { edep = e; }
+    void setTime(double t) { middleTime = t; }
```

Comment:
I recommend removing "setters" to the extent possible--would be better to create a new `SimEnergyDepositLite` object.  This is not always possible, but it is an approach we generally encourage.

Reply:
Done!